### ISSUE TITLE
What to do next with our blazing fast sparse implementation?

### ISSUE THREAD

**Issue by gully** [gully/blase#11]
We now have blazing fast initialization and training of a spectral model, which gives us a ripe environment for innovation: we can do lots of experiments and improvements for relatively cheap.  There are a few steps forward worth thinking about.

### More accurate: Line wings
We covered this in #9 and #10, and it should definitely be done.

### More accurate: True Voigt instead of pseudo-Voigt
Since we're so fast now we can afford to spend more on the previously too-expensive Voigt computation.  We can use the Voigt-Hjerting function, which we have already previously implemented.  It appears to add ~27x the computation time, but should be feasible.

### Parameter studies: accuracy scaling with P_rom
We could make a plot of residual sigma versus Prominence.  At what point do we hit a noise floor from the inaccuracies of our model?

### Parameter studies: Clustering of erroneous line widths
A plot of Amplitude versus gamma width reveals a bunch of impossibly wide lines that are overfitting continuum imperfections

### Application to data
The model accuracy is probably "good enough" to apply to data.  Yay!  Let's do that, since we'll learn a bunch of new things along the way.  We need to implement vsini, etc.

### Continuum flexibility: Polynomial fitting
We could simultaneously fit the continuum polynomial.  We currently have this turned off.

### Continuum flexibility: Gaussian Process
A GP seems like the right way to specify the continuum, but it has a scaling problem.  We could investigate scaling with GPyTorch, or celerite-for-pytorch?

**Comment by gully** [comment]
these themes are all discussed in the paper.  while there's a difference of discussing them versus actually carrying them out, I think it's fine to close this issue and then open new standalone issues for any specific feature we wish to build.
